### PR TITLE
(SIMP-4424) core: Update Puppet to latest Z rel

### PR DESCRIPTION
**Pull request by jeefberkey**
- Update to latest Z release:
  - puppet-agent (1.10.9-1)
  - puppetlabs-release-pc1 (1.1.0-5)
  - puppetserver (2.8.1-1)
- Update cristifalcas-etcd to version with the changes needed (1.12.2)

SIMP-4424 #close

### CODE REVIEW

**Comment by jeefberkey** [comment]
[pipeline status](https://gitlab.com/simp/simp-core/commits/SIMP-4424-puppet-z-update)

File: Puppetfile.tracking [hunk 1]
```diff
@@ -49,12 +49,7 @@ mod 'cristifalcas-journald',
 
 mod 'cristifalcas-etcd',
   :git => 'https://github.com/cristifalcas/puppet-etcd',
```
Comment (lnemsick-simp) [inline Puppetfile.tracking]: wrong url.  Should be simp/puppet-etcd

Comment (jeefberkey) [reply Puppetfile.tracking]: Thanks

**Comment by lnemsick-simp** [comment]
SIMP for CentOS7 ISO builds & bootstraps

**Comment by lnemsick-simp** [comment]
SIMP for CentOS6 ISO builds & bootstraps

**Pull request merged**